cibuildwheel/__main__.py

```diff
@@ -164,7 +183,7 @@ def main() -> None:
     skip_config = options("skip", env_plat=False, sep=" ")
     test_skip = options("test-skip", env_plat=False, sep=" ")
 
-    archs_config_str = options("archs", sep=" ") if args.archs is None else args.archs
+    archs_config_str = args.archs or options("archs", sep=" ")
```

Comment:
Tiny unrelated cleanup - this reads more like the other options, and is clearer - args.archs overrides options, which is rather hidden the way it was written before.